config/boards/README.md

```diff
@@ -14,6 +17,7 @@
 | MODULES_LEGACY | space delimited modules for legacy branch |
 | MODULES_CURRENT | space delimited modules for current branch |
 | MODULES_DEV | space delimited modules for dev branch |
+| MODULES_NEXT | space delimited modules for next branch (deprecated) |
```

Comment:
The same. LEGACY CURRENT DEV

Reply:
Okay, so I remove all the deprecated ones.